packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -424,8 +437,19 @@ Lexis.api = (moduleInstance, store) => {
      * @param {string} lemmaTranslationLang - A locale for lemma translations (e.g. 'en-US')
      * @param wordUsageExamples
      */
-    lookupText: async (textSelector, forWordlist = false) => {
-      return moduleInstance.lexicalQuery({ store, textSelector, source: 'lookup' }, forWordlist)
+    lookupText: async (textSelector) => {
+      return moduleInstance.lexicalQuery({ store, textSelector, source: 'lookup' })
+    },
+
+    /**
+     * Starts a lexical query from a lookup component.
```

Comment:
update comment ("lookup" should be "wordlist")

Reply:
fixed comments